hotel_reservation_summary_range/wizard/hotel_reservation_summary.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from datetime import date, timedelta
+
+from odoo import api, models
+
+
+class RoomReservationSummary(models.TransientModel):
+    _name = "hotel.room.reservation.summary.range"
+    _inherit = "hotel.room.reservation.summary"
+    _form_view = (
+        "hotel_reservation_summary_range"
+        ".hotel_room_reservation_summary_range_form_view"
+    )
+
+    @api.multi
+    @api.depends("date_from", "date_to")
+    def _compute_name(self):
```

Comment:
Seems like this is the same code as in the base module, and so doesn't need to be inherited

Reply:
👍